dist/core.js

```diff
@@ -222,14 +222,18 @@ var extensions = {};
 
 /**
 * Extends core functionalities
-*
-* @method extend
```

Comment:
Instead of removing the docs in here, you should update these to match the new features ;)